x-pack/test/case_api_integration/security_and_spaces/tests/common/cases/reporters/get_reporters.ts

```diff
@@ -23,15 +35,159 @@ export default ({ getService }: FtrProviderContext): void => {
+      const reporters = await getReporters({ supertest: supertestWithoutAuth });
 
-      const { body } = await supertest
-        .get(CASE_REPORTERS_URL)
-        .set('kbn-xsrf', 'true')
-        .send()
-        .expect(200);
+      expect(reporters).to.eql([defaultUser]);
+    });
+
+    describe('rbac', () => {
+      it('User: security solution only - should read the correct reporters', async () => {
+        await createCase(
+          supertestWithoutAuth,
+          getPostCaseRequest({ owner: 'securitySolutionFixture' }),
+          200,
+          {
+            user: secOnly,
+            space: 'space1',
+          }
+        );
+
+        await createCase(
+          supertestWithoutAuth,
+          getPostCaseRequest({ owner: 'observabilityFixture' }),
+          200,
+          {
+            user: obsOnly,
+            space: 'space1',
+          }
+        );
+
+        for (const scenario of [
+          {
+            user: globalRead,
+            expectedReporters: [getUserInfo(secOnly), getUserInfo(obsOnly)],
+          },
+          {
+            user: superUser,
+            expectedReporters: [getUserInfo(secOnly), getUserInfo(obsOnly)],
+          },
+          { user: secOnlyRead, expectedReporters: [getUserInfo(secOnly)] },
+          { user: obsOnlyRead, expectedReporters: [getUserInfo(obsOnly)] },
+          {
+            user: obsSecRead,
+            expectedReporters: [getUserInfo(secOnly), getUserInfo(obsOnly)],
+          },
+        ]) {
+          const reporters = await getReporters({
+            supertest: supertestWithoutAuth,
+            expectedHttpCode: 200,
+            auth: {
+              user: scenario.user,
+              space: 'space1',
+            },
+          });
+
+          expect(reporters).to.eql(scenario.expectedReporters);
+        }
+      });
+
+      for (const scenario of [
+        { user: noKibanaPrivileges, space: 'space1' },
+        { user: secOnly, space: 'space2' },
+      ]) {
+        it(`User ${scenario.user.username} with role(s) ${scenario.user.roles.join()} and space ${
+          scenario.space
+        } - should NOT get all reporters`, async () => {
+          // super user creates a case at the appropriate space
+          await createCase(
+            supertestWithoutAuth,
+            getPostCaseRequest({ owner: 'securitySolutionFixture' }),
+            200,
+            {
+              user: superUser,
+              space: scenario.space,
+            }
+          );
+
+          // user should not be able to get all reporters at the appropriate space
+          await getReporters({
+            supertest: supertestWithoutAuth,
+            expectedHttpCode: 403,
+            auth: { user: scenario.user, space: scenario.space },
+          });
+        });
+      }
+
+      it('should respect the owner filter when having permissions', async () => {
+        await Promise.all([
+          await createCase(
```

Comment:
I think we can remove the `await` next to the calls within the `Promise.all` right?